code/modules/mob/living/simple_animal/borer/egg.dm

```diff
@@ -98,6 +98,9 @@
 	else
 		..()
 
+/obj/item/weapon/reagent_containers/food/snacks/borer_egg/attack_ghost(var/mob/dead/observer/O)
+	visible_message(message = "<span class='notice'>\The [src] wriggles vigorously.</span>", blind_message = "<span class='danger'>You hear what you think is someone jiggling a jelly.</span>")
```

Comment:
Cooldowns on attacks are hot spaghetti and shouldn't be relied on. Like half the things have them and the ones that DO have them do more harm than good.